src/main/java/frc/robot/subsystems/conveyor/ConveyorIOSim.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.subsystems.conveyor;
+
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+import edu.wpi.first.wpilibj.Timer;
+import lib.motors.SparkMaxSim;
+
+import static frc.robot.subsystems.conveyor.ConveyorConstants.*;
+
+public class ConveyorIOSim implements ConveyorIO {
+    private final SparkMaxSim conveyor;
+
+    public static PIDController controller = new PIDController(KP.get(), KI.get(), KD.get(), Timer.getFPGATimestamp());
```

Comment:
If you want to use the "tunable" feature, you'll need to update these values sometimes in run-time. See 6328's 2023 code for usage

Reply:
in the meantime we just set a default value

Reply:
7ac5c8b8c2c167f2abb3c32174632f64b4964dbf

Reply:
@vichik123 is that ok? 
For the real implementation you will update the value?